AudioPlayer.h

```diff
@@ -21,7 +21,15 @@
 - (void)play;
 - (void)resume;
 - (BOOL)isPlaying;
-- (double)getTotalTime;
-- (double)getCurrentTime;
+- (CMTime)getTotalTime;
+- (CMTime)getCurrentTime;
+- (void)seekToTime:(CMTime)time;
+- (void)seekToTimeWithTolerance:(CMTime)time;
+- (BOOL)compareURL:(NSURL *)myURL;
+- (void)receiveTitle:(NSString*)myTitle;
```

Comment:
This name is confusing. Can you change the name of this function and the variable?